sql/Pg-database.sql

```diff
@@ -1466,6 +1466,7 @@ CREATE TABLE acc_trans (
   entry_id SERIAL PRIMARY KEY
 );
 
+-- Drop this, no longer used
 ALTER TABLE cr_report ADD FOREIGN KEY (max_ac_id) REFERENCES acc_trans(entry_id);
```

Comment:
max_ac_id was there to ensure that it wouldn't take 100 years to find the new entries and because the original author apparently thought it a safe criterion. It's highly unlikely that the criterion ever was actually safe, but it's very likely it worked in practice.

When removing the cache, we really need to introduce a different criterion (index, maybe?) which prevents the full transactions table from being scanned for each reconciliation. We can't *just* remove one thing and not replace it by another.